src/test/unit/lang/stanc_helper_test.cpp

```diff
@@ -28,14 +28,14 @@ TEST(commandStancHelper, printStancHelp) {
 
 bool create_test_file(const std::string& path, const std::string& program) {
   std::string cmd_setup = "mkdir -p test/test-models";
-  system(cmd_setup.c_str());
+  int return_code = system(cmd_setup.c_str());
   std::string cmd = "echo ";
   cmd += "\"";
   cmd += program;
   cmd += "\"";
   cmd += " > ";
   cmd += path;
-  int return_code = system(cmd.c_str());
+  return_code = system(cmd.c_str());
```

Comment:
If you dont want to actually ignore errors in the first call:
```suggestion
  return_code |= system(cmd.c_str());
```

Reply:
I will just cat to void. The return code was not used before, so that is the least invasive change.

Reply:
Done.